### PR TITLE
Android DnD

### PR DESCRIPTION
**Pull request by bhollis**
https://github.com/react-dnd/react-dnd/issues/3544 indicates this may be a React-DnD problem, but that library is mostly unmaintained at this point...

### CODE REVIEW

**Comment by robojumper** [comment]
Will test on my phone the moment the API comes back.

File: src/app/utils/browsers.ts [hunk 1]
```diff
@@ -32,4 +32,16 @@ export function isMac() {
   return mac;
 }
 
-export const isNativeDragAndDropSupported = () => 'draggable' in document.createElement('div');
+const android = navigator.platform.includes('Android');
+export function isAndroid() {
+  return android;
+}
```
Comment (robojumper) [inline src/app/utils/browsers.ts:38]: On my Android 11 with Firefox `navigator.platform` evaluates to `Linux aarch64` and with Chrome it evaluates to `Linux armv8l`, so this should probably test for `Linux` instead. That way drag and drop works on Chrome, but I can't get it to work on Firefox.

Comment (bhollis) [reply src/app/utils/browsers.ts:38]: Maybe I could test `userAgent` against `Android` then?

Comment (robojumper) [reply src/app/utils/browsers.ts]: That'd also work

Comment (robojumper) [reply src/app/utils/browsers.ts]: Actually, not sure if this is a common use case, but when people toggle the "Desktop site" setting in Firefox or Chrome on Android then the browsers spoof their User Agent to not include "Android", and in that case dnd won't work.

Comment (bhollis) [reply src/app/utils/browsers.ts]: That's a good point, but I'd rather be able to say "you shouldn't request the desktop site" than have things fail to work in Linux for some reason. Feels safe enough to mostly copy the logic at https://github.com/timruffles/mobile-drag-drop/blob/master/src/internal/feature-detection.ts

**Comment by 55Cancri** [comment]
@bhollis @robojumper 

have you guys seen any update on this or managed to get this to work? The react-dnd example still does not work for me on chrome android. When I try to drag, it starts the drag because the item disappears, but thats it, then the page just scrolls.

**Comment by bhollis** [comment]
No, we haven't tried any further to make native DnD work on Android

**Pull request merged**